pom.xml

```diff
@@ -19,6 +19,7 @@
 
     <properties>
         <maven.compiler.release>17</maven.compiler.release>
+        <maven.compiler.parameters>true</maven.compiler.parameters>
```

Comment:
Even with this, there is still a weird HV warning, I will need to have a look at it.